_data/3.7-drivers.yml

```diff
@@ -123,6 +123,8 @@
           href: spring-data-migration-migrating-1-x-3-0.html
         - text: Migrating 2.x to 3.0
           href: spring-data-migration-migrating-2-x-3-0.html
+- text: ArangoDB Spark Connector New
```

Comment:
To avoid confusion in the names, we could name this library `ArangoDB Spark Datasource` and always use this name when referring to it.
WDYT?